src/styles/isomer-cms/pages/Editor.module.scss

```diff
@@ -3,6 +3,43 @@
 @import "../elements/variable";
 
 .preview-styles {
+  // NOTE: These css variables all come from sgds.
+  // This has to be done because the css variables will
+  // otherwise be undefined at runtime as
+  // they are not loaded by sass.
```

Comment:
why are they not loaded?

Reply:
honestly i'm not sure too. my initial thinking was that this was because of `load_css` rather than `use` (see [here](https://sass-lang.com/documentation/modules/meta/#load-css)) but using `@use` has the same issue ._.

Reply:
oddly enough, using an old version of `master` does **not** have this issue. however, given that design system doesn't actually impact this, i think this might be caused by deps upgrade (of `sass` possibly) leading to behaviour change ._.

Reply:
Oh, seems like that might be the root issue instead we should be solving for? But, am okay to merge this for now